src/main/java/nl/talsmasoftware/umldoclet/uml/util/JavaBeanProperty.java

```diff
@@ -51,6 +51,9 @@ public class JavaBeanProperty {
     private Method getter;
     private Method setter;
 
+    private static final int startIndexGetSet = 3;
```

Comment:
Constants should be before fields.
Constant names should be all-uppercase characters separated by udnerscores.